aws_security_group.tf

```diff
@@ -112,296 +112,296 @@ resource "aws_security_group" "acme_web" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 23
     to_port     = 23
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 25
     to_port     = 25
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure no security groups allow ingress from 0.0.0.0/0 to ALL ports and protocols**
 
**Vulnerability**
 It is recommended that no security group allows unrestricted ingress access
 
**Remediation**
Reduce the scope of the inbound rules to just the necessary scope, protocol, and ports

source: accurics-